### PR TITLE
=doc Consolidate client side index page #1290

### PR DESCRIPTION
**Pull request by jlprat**
Issue: #1290

### CODE REVIEW

File: docs/src/main/paradox/scala/http/client-side/index.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ Depending on your application's specific needs you can choose from three differe
   the user can supply a ``Source[HttpRequest, NotUsed]`` with requests to run against a single host
```
Comment (jonas) [inline docs/src/main/paradox/scala/http/client-side/index.md]: Drop the double back-ticks and add a `@java[Source<HttpRequest, NotUsed>]` (not sure whether Java uses `NotUsed` or `Void`).

File: docs/src/main/paradox/scala/http/client-side/index.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Depending on your application's specific needs you can choose from three differe
...
   over multiple pooled connections.
 
-@ref[Connection-Level Client-Side API](connection-level.md#connection-level-api)
+@ref[Connection-Level Client-Side API](connection-level.md)
 : for full-control over when HTTP connections are opened/closed and how requests are scheduled across them. Only
```
Comment (jonas) [inline docs/src/main/paradox/scala/http/client-side/index.md]: `s/full-control/full control/`

Comment (jlprat) [inline docs/src/main/paradox/scala/http/client-side/index.md]: Agree

File: docs/src/main/paradox/scala/http/client-side/index.md (hunk 1 above)
Comment (jlprat) [inline docs/src/main/paradox/scala/http/client-side/index.md]: Will check which is the right one

**Comment by ktoso** [comment]
LGTM thanks :)

**Pull request merged**